cmd/config.go

```diff
@@ -41,6 +41,9 @@ type Config struct {
 	SnapshotBlockHeightPeriod uint64
 	DisableEncoderMigrations  bool
 
+	// Mempool
+	BatchPersistFrequencyMilliseconds uint64
```

Comment:
This is a mempool param that's defined outside of the mempool. It's good to contextualize by prefixing the name
```suggestion
	MempoolBatchPersistFrequencyMilliseconds uint64
```